src/main/java/seedu/duke/Trip.java

```diff
@@ -82,4 +83,18 @@ public void setPersonName(int indexOfPerson, String newName) {
         personToEdit.setName(newName);
     }
 
+    public void addExpense(Expense expense) {
+        listOfExpenses.add(expense);
+    }
+
+    public void removeExpense(Expense expense) {
+        listOfExpenses.remove(expense);
+    }
```

Comment:
May need to update this in future iterations to delete using index instead of object (no action required)